tests/updates/sparse_update_test.py

```diff
@@ -0,0 +1,35 @@
+"""Tests for our sparse Update."""
+
+
+import numpy as np
+import pytest
+from git_theta import async_utils
+from git_theta import params
+from git_theta.updates import sparse
+import scipy.sparse
+
+SHAPE = 100
+NUM_UPDATES = 100
+TRIALS = 50
+
+
+@pytest.fixture
+def updater():
+    return sparse.SparseUpdate(params.get_update_serializer())
+
+
+def test_sparse_update_application(updater):
```

Comment:
probably rename this to something like round trip as it tests both the calculate update and the apply

Reply:
Got it, renamed the function